tests/integrations/test_keras.py

```diff
@@ -89,6 +86,7 @@ def test_basic_keras(dummy_model, dummy_data, wandb_init_run):
     # NOTE: backend mock doesnt copy history into summary (happens in internal process)
     # assert wandb.run._backend.summary["loss"] > 0
     assert len(graph_json(wandb.run)["nodes"]) == 3
+    wandb.finish()
```

Comment:
why do you need to add finish here?

Reply:
I think we should explicitly finish things up and not be relying on the `test_settings` fixture that will do it at teardown.

Reply:
well.. technically it is relying on `wandb_init_run` which is also doing the init for you, i think it makes sense that if you are using a fixture to do the init you can assume it will do the finish for you.